### ISSUE TITLE
MobileNetV3 Small/Large not fully quantized after full integer post-training quantization runs with no errors

### ISSUE THREAD

**Issue by drajsel** [tensorflow/tensorflow#43257]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): no
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): N/A (used Google Colaboratory)
- TensorFlow version (using the command in the template): v1.12.1-39569-gcb34190201 2.4.0-dev20200818

**Describe the current behavior**
The model is not fully quantized after full integer quantization even though no error occurs during the quantization. 

**Describe the expected behavior**
I tried using the quantized version in the example Android app but it crashed every time so I wanted to see if the quantized version worked on Coral devices, but then it failed to compile with the error "Model is not fully quantized.".

**Standalone code to reproduce the issue**
https://colab.research.google.com/drive/1B-VbTOCpmGhgv5adOpnGn68VARo30bF9?usp=sharing

**Comment by ravikyram** [comment]
I could reproduce the issue in colab with TF nightly version(`2.4.0-dev20200915`) .Please, find the gist [here](https://colab.research.google.com/gist/ravikyram/d303b7aebb1a4c8fef8743902cb0153e/untitled361.ipynb). Thanks!

**Comment by padmasreenagarajan** [comment]
Hi..Im trying to convert Mobilenetv3 onnx model to tensorflow.
I need tensorflow==1.12.0 for my inferencing.
I searched all over to find the onnx and onnx-tf versions corresponding to tensorflow==1.12.0

Could anyone suggest me the compatible versions of onnx,onnx-tf and torch for my required tensorflow version?
@ravikyram @ymodak @drajsel 
Thanks in advance!

**Comment by bhack** [comment]
@drajsel @ravikyram You need to use the old TOCO with `converter.experimental_new_converter = False` for a working coral compilation. 
This is cause:
https://github.com/google-coral/edgetpu/issues/168#issuecomment-679233152
/cc @Namburger

**Comment by drajsel** [comment]
@ravikyram @bhack The compilation is successful now, but it doesn't seem to matter now because of this:
```
Number of operations that will run on Edge TPU: 1
Number of operations that will run on CPU: 122
```
Thank you all anyway! 

It worked for the downloaded model, but in the case where I have two outputs it doesn't work (`Unsupported explicit zero output index: serving_default_input_2:0`). Do you have any suggestions, what I should look into? I know it's outside the scope of this bug, but I've tried many approaches in TF 2, TF 1.15, using the toco command line converter like seen in quantization for object detection models in TF 1.x etc.

**Comment by bhack** [comment]
@drajsel I don't know If there Is something fresh from the unstable apt cannel instead of the stable in echo `"deb https://packages.cloud.google.com/apt coral-edgetpu-stable main" | sudo tee /etc/apt/sources.list.d/coral-edgetpu.list`
So that you don't need to use TOCO (converter.experimental_new_converter = False)

**Comment by vikramdattu** [comment]
@djalusic I know this is quite an old topic.
I saw you are converting the model with usual way. Did you get quantised model accuracy anywhere close to the original float model?
https://colab.research.google.com/drive/1B-VbTOCpmGhgv5adOpnGn68VARo30bF9?usp=sharing 
Can you share some ideas what worked for you?

**Comment by djalusic** [comment]
@vikramdattu I must admit I forgot what were the results with this model, I just remember that I didn't use this model in the end, maybe mobilenetv2. I'm sorry I can't be more helpful.
